docs/variants/protectli_vp66xx/releases.md

```diff
@@ -12,7 +12,43 @@ For details about our release process please read
 </center>
 
 Test results for this platform can be found
-[here](https://docs.google.com/spreadsheets/d/1wI0qBSLdaluayYsm_lIa9iJ9LnPnCOZ9eNOyrKSc-j4/edit#gid=1316498194).
+[here](https://github.com/Dasharo/osfv-results/blob/main/boards/Protectli/VP66xx/).
```

Comment:
```suggestion
[here](https://github.com/Dasharo/osfv-results/blob/main/boards/Protectli/VP66xx/v0.9.0-results.csv).
```

Reply:
Then the link is duplicated. 
That is supposed to be a link to results of the platform - *The results for this platform (...)*. `v0.9.0-results.csv` contains results of a single release, it is linked below as *Test results for this release (...)*

Reply:
Yes, either put a link to the spreadsheet directly, but the whole sentence has to be under given release section, or give the link to the directory with all results at the top of the file or have both

Reply:
I see we have both, os it is fine.